Detectors/FIT/FDD/reconstruction/src/test-raw-conversion.cxx

```diff
@@ -35,8 +35,8 @@ int main(int argc, char* argv[])
   }
   std::vector<o2::fdd::Digit> digitsBC, *fddBCDataPtr = &digitsBC;
   std::vector<o2::fdd::ChannelData> digitsCh, *fddChDataPtr = &digitsCh;
-  tree->SetBranchAddress("FDDDIGITSBC", &fddBCDataPtr);
-  tree->SetBranchAddress("FDDDIGITSCH", &fddChDataPtr);
+  tree->SetBranchAddress("FDDDigit", &fddBCDataPtr);
+  tree->SetBranchAddress("FDDDigitCh", &fddChDataPtr);
```

Comment:
I'm not sure if you can change these just like that. These are the names of branches set in your digitizer (or any piece of code that produces digits, also raw converter). If you want to change it, you need to do it in all places

Reply:
This is not necessary at the moment. I will revert these changes. Thank you.